### ISSUE TITLE
`npm update` is changing package.json.

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/chipper#1035]
`npm update` is consistently changing package.json.

Here's an example for chipper, but I see the same behavior for any repo.

```
% node --version
v14.15.0
% npm --version
7.9.0
% cd chipper
% npm prune
...
% git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
% npm update
...
% git status
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   package.json

no changes added to commit (use "git add" and/or "git commit -a")
%  git diff package.json 
diff --git a/package.json b/package.json
index 5c222d3e..8bd15e71 100644
--- a/package.json
+++ b/package.json
@@ -7,13 +7,14 @@
     "url": "https://github.com/phetsims/chipper.git"
   },
   "devDependencies": {
-    "archiver": "~3.1.1",
     "@babel/core": "~7.8.6",
     "@babel/preset-env": "~7.8.6",
+    "archiver": "~3.1.1",
     "babel-eslint": "^10.0.3",
     "eslint": "~7.22.0",
     "eslint-plugin-react": "^7.18.0",
     "grunt": "~1.1.0",
+    "html-webpack-plugin": "4.0.0-beta.11",
     "jimp": "^0.2.0",
     "jpeg-js": "~0.4.3",
     "jsdoc": "~3.6.3",
@@ -21,6 +22,7 @@
     "lodash": "^4.17.10",
     "marked": "^0.3.13",
     "md5": "~2",
+    "memfs": "^3.0.3",
     "node-html-encoder": "~0.0.2",
     "pngjs": "~0.4.0",
     "puppeteer": "^5.4.1",
@@ -31,9 +33,7 @@
     "terser": "~4.6.4",
     "webpack": "^4.41.4",
     "webpack-cli": "^3.3.10",
-    "webpack-dev-server": "^3.10.1",
-    "html-webpack-plugin": "4.0.0-beta.11",
-    "memfs": "^3.0.3"
+    "webpack-dev-server": "^3.10.1"
   },
   "eslintConfig": {
     "extends": "../chipper/eslint/sim_eslintrc.js",
```

**Comment by pixelzoom** [comment]
@jonathanolson mentioned that @mattpen had a similar problem, reported in https://github.com/npm/cli/issues/3044.

@samreid pointed out that the current versions of Node.js are:
> Latest LTS Version: 14.16.1 (includes npm 6.14.12)
> Latest Current Version: 15.14.0 (includes npm 7.7.6)

I have no idea why I have npm 7.9.0 with node v14.15.0.

I downloaded and installed LTS 14.16.1 from https://nodejs.org/en/.  Now I have node 14.16.1 and npm 6.14.12.  And the problem seems to have gone away.

I'll self-assign and keep this open until I've deployed an RC, to verify that there are no problems with tasks that run `npm update`.
